data/transactions/logic/assembler.go

```diff
@@ -2338,20 +2437,6 @@ func checkByteConstBlock(cx *EvalContext) error {
 	return nil
 }
 
-func disIntcblock(dis *disassembleState, spec *OpSpec) (string, error) {
-	intc, nextpc, err := parseIntcblock(dis.program, dis.pc)
-	if err != nil {
-		return "", err
-	}
-	dis.nextpc = nextpc
-	out := spec.Name
-	for _, iv := range intc {
-		dis.intc = append(dis.intc, iv)
-		out += fmt.Sprintf(" %d", iv)
-	}
-	return out, nil
-}
-
 func disIntc(dis *disassembleState, spec *OpSpec) (string, error) {
```

Comment:
@jannotti Friendly pointer to unused method:  `disIntc`.  I believe it's _safe_ to remove it.